content/en/docs/reference/glossary/endpoint.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Endpoints
+id: endpoints
+date: 2020-04-23
+full_link: 
```

Comment:
Why is this blank?

Either:
- omit the field and create an issue to create a page that it can link to
- create that page (eg: https://kubernetes.io/docs/concepts/services-networking/endpoints/) and link to it.

Reply:
I took reference from other items in the glossary that lacks a full_link. It was left blank instead of being omitted. (e.g App-Container, PodPreset, Service Broker). I created a new issue as suggested #21959.